src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -163,7 +174,11 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             app.UseResponseCompression();
-            app.UseMvc();
+
+            app.UseEndpoints(builder =>
```

Comment:
Can you make sure our route restrictions that prevent sensitive endpoints on the metrics port still work as expected?

Reply:
Seems to be working correctly: I can get to /processes and /metrics on 52323 but only to /metrics on 52325.